### PR TITLE
Fix checkout

### PR DESCRIPTION
**Pull request by moretea**
These changes make it possible to check out the cart again. We ran into this problem at the FP hackathon.

It still crashes if you're not logged in.

### CODE REVIEW

File: api/orders/index.js [hunk 1]
```diff
@@ -64,6 +64,12 @@
             }
             console.log("Received response: " + JSON.stringify(body));
             var jsonBody = JSON.parse(body);
+
+            if (jsonBody._links == undefined) {
+              callback(error);
```
Comment (vlal) [inline api/orders/index.js]: I don't think error will have anything since it's suppose to be caught by the first if, no?

Comment (moretea) [inline api/orders/index.js]: Yes, this is wrong. It prevented the nodejs process from crashing at least.

**Comment by moretea** [comment]
Closed in favor of https://github.com/microservices-demo/front-end/pull/49

File: api/orders/index.js (hunk 1 above)
Comment (moretea) [inline api/orders/index.js]: I'm closing this issue, since #49 solves just this problem. I'll investigate the nodejs crashes tomorrow.

**Pull request closed without merging**